src-graphql/collection-query.js

```diff
@@ -0,0 +1,36 @@
+import imageQuery from './image-query';
+import createGid from './create-gid';
+
+export default function collectionQuery({client, id}, fields = ['id', 'handle', 'updatedAt', 'title', ['image', imageQuery()]]) {
+  let query;
+
+  if (id) {
+    query = client.query((root) => {
+      root.add('collection', {args: {id: createGid('Collection', id)}}, (collection) => {
```

Comment:
there's no longer a root `collection` field. Should use the node interface and an inline fragment.